docs/user-docs/latest/media-library/organizing-assets-with-folders.md

```diff
@@ -10,8 +10,8 @@ Folders in the Media Library help you organize uploaded assets. Folders sit at t
 
 From the Media Library, it is possible to view the list of folders and browse a folder's content, create new folders, edit an existing folder, move assets to a folder, and delete a folder.
 
-::: caution
-Downgrading from Strapi v4.3.0-beta to v4.2.0 will delete folders of the Media Library. Even though assets should be preserved, we recommend you test beta versions of Strapi on a new project rather than on an existing one, as unexpected consequences could impact your database.
+::: note
+Folders follow the permission system of assets (see [Users, Roles & Permissions](/user-docs/latest/users-roles-permissions/introduction-to-users-roles-permissions.md)). You can't define specific permissions for a folder.
```

Comment:
Wouldn't it be better to follow passive voice here?

```suggestion
Folders follow the permission system of assets (see [Users, Roles & Permissions](/user-docs/latest/users-roles-permissions/introduction-to-users-roles-permissions.md)). It is not yet possible to define specific permissions for a folder.
```

Reply:
@gu-stav Strictly speaking, yes, it would be more compliant with our style guide, but FYI we are a bit more liberal with the voicing in callouts — which are almost the only location where we are more personal and use "you".

I'll apply your suggestion as it makes a lot of sense here :-)